ui-events/src/pointer/mod.rs

```diff
@@ -214,6 +236,40 @@ impl PointerUpdate {
     }
 }
 
+/// A relative pointer motion update.
+#[derive(Clone, Debug, PartialEq)]
+pub struct PointerRelativeMotion {
+    /// Identifying information about pointer.
+    pub pointer: PointerInfo,
+    /// Cumulative motion over the frames.
+    ///
+    /// The timestamp is for the last integrated frame.
+    pub total: PointerRelativeFrame,
+    /// All integrated frames in this motion update.
+    ///
+    /// These primarily come from event coalescing.
+    /// Coalescing is application-specific.
+    /// On the web, the browser does its own coalescing, whereas
+    /// on other platforms you may do your own, or forego it
+    /// altogether, delivering every frame.
+    ///
+    /// This includes the most recent frame.
+    pub frames: Vec<PointerRelativeFrame>,
```

Comment:
This isn't the same definition of "frame" that we use everywhere ... not sure about this.

Reply:
I could use the word ‘sample’ instead. It's not exactly a state since it includes a delta along with several state parameters (buttons, click count, etc.)